crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -272,6 +313,18 @@ pub struct StakingPool {
     pub pending_pool_token_withdraw: u64,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+pub struct PoolTokenExchangeRate {
+    sui_amount: u64,
+    pool_token_amount: u64,
+}
+
+impl PoolTokenExchangeRate {
+    pub fn rate(&self) -> f64 {
```

Comment:
Might be worth adding a doc comment explaining which amount is the numerator / denominator.